mava/configs/system/q_learning/rec_iql.yaml

```
@@ -0,0 +1,33 @@
+# --- Defaults REC-IDQN ---
+total_timesteps: 5_000_000
+num_updates: 1000 # total_timesteps takes preference
+seed: 1
+
+# --- Agent observations ---
+add_agent_id: True
+
+# --- RL hyperparameters ---
+buffer_min_size: 32
+update_batch_size: 1 # Number of vectorised gradient updates per device.
+
+rollout_length: 2 # Number of environment steps per vectorised environment.
+epochs: 2 # Number of learn epochs per training data batch.
+
+# sizes
+buffer_size: 5_000  # size of the replay buffer. Note: total size is this * num_devices
```

Same here for good default. I think even though the buffers are sharded we'd want a big buffer still, although if you've seen different results in experiments let me know. Just 5k is very small
```suggestion
buffer_size: 1_000_000  # size of the replay buffer. Note: total size is this * num_devices
```

---

5k is indeed very small - in my experience with SMAX, rbs on the order of 100k are unable to solve some of the scenarios, where rbs on the order of 5k or 3k can. JAXMARL uses 3k, which is maybe a bit small, but I found out by experience that it just seems to have a greater learning capacity that way.